src/domain/flow-system/domain/src/entities/shared/flow_run_rule.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use crate::*;
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Default, Debug, Clone, PartialEq, Eq)]
+pub struct ConfigSnapshot {
```

Comment:
Confusing name. `FlowConfigSnapshot` maybe. Dima is doing very different kind of configs, i.e.